Dangerfile

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+github.dismiss_out_of_range_messages
```

Comment:
YES! That was something about Hound that bothered me so much (it keeping the messages from violations that had since been fixed)…